.circleci/config.yml

```diff
@@ -0,0 +1,42 @@
+version: 2
+jobs:
+  build:
+    parallelism: 3 # run three instances of this job in parallel
+    docker:
+      - image: circleci/ruby:2.4.2-jessie-node
```

Comment:
2.4 is the [oldest supported version](https://en.wikipedia.org/wiki/Ruby_(programming_language)#Table_of_versions)